DDFIngestTest.py

```diff
@@ -0,0 +1,67 @@
+import sys, requests, re, time
+
+def main():
+    if len(sys.argv) != 4:
+        print_usage_statement()
+        return
+
+    ingest_type = sys.argv[1]
+    file_path = sys.argv[2]
+    xml_type = sys.argv[3]
+
+    with open(file_path, "r") as file:
+        file_contents = file.read()
+
+    requests.packages.urllib3.disable_warnings()
+    headers = {
+        "Content-Type" : "application/xml",
+        "Authorization" : "Basic YWRtaW46YWRtaW4="
+    }
+
+    url = "https://localhost:8993/services/csw/"
+
+    if ingest_type == "csw":
+        file_contents = generate_csw_transaction_xml(file_contents, xml_type)
+    elif ingest_type == "rest":
+        url = "https://localhost:8993/services/catalog?transform=" + xml_type
+    else:
+        print_usage_statement()
+        return
+
+    start = time.time()
+    response = requests.post(url, file_contents, headers=headers, verify=False)
+    elapsed = time.time() - start
+    num_inserted = get_number_of_inserted_records(response, ingest_type)
+
+    print("{} record(s) inserted in {} seconds.".format(num_inserted, round(elapsed, 2)))
```

Comment:
Should we round to 3, milliseconds seems to be more common for these types of things, plus 5 milliseconds may be significant if a very small set of records are ingested